jax/experimental/jax2tf/primitives_with_limited_support.md

```diff
@@ -29,3 +39,29 @@
 | scatter-mul | Missing TF support | scatter-mul is unimplemented for dtype complex64 | TPU |
 | select_and_gather_add | Missing TF support | select_and_gather_add is unimplemented for dtype float32 | TPU |
 | svd | Missing TF support | svd is unimplemented for dtype complex64; this works on JAX because JAX uses a custom implementation | CPU, GPU |
+
+## Not yet implemented primitive conversions
+
```

Comment:
I would add some text here as well:

The conversion of the following JAX primitives is not yet implemented. 

Nit: I would either put them all in one line, or put them in a table with similar format as the primitives above.

Reply:
Added & put all the primitives on one line.